server/services/alert.js

```diff
@@ -12,10 +14,13 @@ function createAlert(ticket) {
     .then((user) => {
       const firstDevice = user.devices[0];
       sendPage(ticket, user, firstDevice);
-    });
+    })
+    .then(() => ticketService.addAction(ticket.id, actionTypes.CREATED));
```

Comment:
I think once the `createAlert()` is refactored to work with the new queue it should be written. For right now I know it's about to change next week, so it seems pointless.